mist-invoice-contracts/.gitignore

```diff
@@ -1,6 +1,7 @@
 # Compiler files
 cache/
 out/
+lib/**
```

Comment:
This snuck in, sorry. I was getting foundry errors and did this because I didn't understand the error. My bad on this.